index.html

```diff
@@ -102,9 +102,124 @@ <h2>Work</h2>
+                 </div>
+  
+             </div>
+             <div class="box-1 row__rev">
+                 <img src="./images/ImgPlaceholder.png" alt="Blank square">
+                 <div class="box-2">
+                    <h1>Multi-Post Stories</h1>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul>
+                        <li>css</li>
+                        <li>Html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+     
+                    </ul>
+                    <button type="button" class="button">See Project</button>
+                 </div>
+             </div>
+          </article>
+        </div>
+        <section class="contact-form">
+            <h3>Let's chat!</h3>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p> 
+            <form class="form" action="https://formspree.io/f/xoqrbwdq" method="POST">
+                <div class="form-1">
+                    <div>
+                        <label for="name">
+                            <input id="name" type="text" placeholder="Yannick" required minlength="6" maxlength="30">
+                        </label>
+                    </div>
+                    <div>
+                        <label for="text">
+                            <input id="text" type="text" placeholder="Last name" required minlength="6" maxlength="30">
+                        </label>
+                    </div>
+                </div>
+                <div>
+                    <label for="email">
+                        <input id="email" type="email" placeholder="yannickmugisha23@gmail.com" required>
+                    </label>
+                    
+                </div>
+                <div>
+                    <label for="textarea">
+                        <textarea id="textarea" name="textarea" cols="50" rows="8" placeholder="Enter text here" minlength="15" maxlength="500"></textarea>
+                    </label>
+                    
+                </div>
+                <div class="down-form">
+                    <button type="submit">Send</button>
+                   <div class="social-icons">
+                    <div class="left">
+                        <ul>
+                            <li><i class="fa-brands fa-github"></i></li>
+                            <li><i class="fa-brands fa-linkedin-in"></i></li>
+                            <li><i class="fa-brands fa-angellist"></i></li>
+                            <li><i class="fa-brands fa-twitter"></i></li>
+                            <li><i class="fa-solid fa-m"></i></li>
+                        </ul>
+                    </div>

```

Comment:
- [ ] Kindly align these to the center, as per the design in the Figma template, they are currently aligned to the left:

![image](https://user-images.githubusercontent.com/20567503/185741811-cf36179b-c1d6-4fc0-87e5-770eb71b58c0.png)